<?xml version="1.0"?>
<launch>
  
  <include file="$(find tiago_iaslab_simulation)/launch/start_simulation.launch">
        <arg name="world_name" value="ias_lab_room_full_tables" />
  </include>
  
  <include file="$(find tiago_iaslab_simulation)/launch/apriltag.launch" />

  <include file="$(find tiago_iaslab_simulation)/launch/navigation.launch" />
  
  <node name="human_node" pkg="tiago_iaslab_simulation" type="human_node" output="screen" />
  
  <node name="MyActionServer" pkg="tiago_iaslab_simulation" type="MyActionServer" output="screen" />
    
  
</launch>
